docs/events_handling.md

```diff
@@ -0,0 +1,89 @@
+
+# Events handling
+
+## Subscribed events
+
+Precaution needs to subscribe to a number of events when it's
+to satisfy different use cases of our users.
+
+The events can be separated into two groups `pull requests
+events` and `check suites/runs events`.
+
+### Pull requests events
+
+Official GitHub documentation: [PullRequest event](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
```

Comment:
```suggestion
Official GitHub documentation: [Pull Request event](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
```

Reply:
The event name in the GitHub docs is `PullRequestEvent`, I favour using that as the link title

Reply:
I agree with Joshua.